web/packages/test/scripts/configure-substrate.sh

```diff
@@ -85,6 +85,16 @@ config_xcm_version() {
     send_governance_transact_from_relaychain $ASSET_HUB_PARAID "$call"
 }
 
+config_penpal() {
+    # System::set_storage(CustomizableAssetFromSystemAssetHub,Ethereum)
+    local call=0x00040440770800eb78be69c327d8334d0927607220020109079edaa802
+    send_governance_transact_from_relaychain $PENPAL_PARAID "$call"
+
+    # foreignAssets::force_create(WETH)
+    local call=0x3301020209079edaa802030087d1f7fdfee7f651fabc8bfcb6e086c278b77a7d00d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0104
+    send_governance_transact_from_relaychain $PENPAL_PARAID "$call"
+}
+
 configure_substrate() {
     set_gateway
```

Comment:
```suggestion
    config_penpal &
    set_gateway
```

Reply:
Penpal is not allowed to execute sudo operation through xcm, it's a test chain which has the sudo pallet. 

Explicit the call here but not execute only for a hint.